src/aics-image-viewer/components/App/types.ts

```diff
@@ -1,8 +1,48 @@
 import { View3d, Volume, ImageInfo } from "@aics/volume-viewer";
-import { ImageType, ViewMode } from "../../shared/enums";
+import { ImageType, RenderMode, ViewMode } from "../../shared/enums";
+import { PerAxis } from "../../shared/types";
 import { ColorArray } from "../../shared/utils/colorRepresentations";
 import { ChannelGrouping, ChannelState, ViewerChannelSettings } from "../../shared/utils/viewerChannelSettings";
 
+type ControlNames =
+  | "alphaMask"
+  | "autoRotateButton"
+  | "axisClipSliders"
+  | "brightnessSlider"
+  | "colorPicker"
```

Comment:
what's "colorPicker"?  I can't think of what this is for

Reply:
It's for the "customize" section in "advanced settings," which contains only color pickers. Could probably be split into `backgroundColorPicker` and `boundingBoxColorPicker`

Reply:
hmm... and it doesn't appear to be hooked up to anything anyways. Interesting.